lisa/tools/iperf3.py

```diff
@@ -47,59 +53,258 @@ def install(self) -> bool:
             self._install_from_src()
         return self._check_exists()
 
-    def run_as_server(self, daemon: bool = True) -> None:
+    def run_as_server_async(
+        self,
+        port: int = 0,
+        format: str = "",
+        interval_seconds: int = 0,
+        output_json: bool = False,
+        one_off: bool = False,
```

Comment:
add comments to explain new parameters.

Reply:
one_off is ==> -1:  handle one client connection then exit

Reply:
update